src/main/java/uk/ac/ebi/ena/webin/cli/context/reads/ReadsManifestReader.java

```diff
@@ -136,8 +134,8 @@ public ReadsManifestReader(
                 ,
                 // File groups.
                 new ManifestFileCount.Builder()
-                        .group("Single or paired sequence reads in one or two fastq files.")
-                        .required(Field.FASTQ, 2)
+                        .group("Single or paired sequence reads in multiple files.")
+                        .required(Field.FASTQ, 999)
```

Comment:
Thought about this more this morning and I suggest that we cap the number Fastq files to 10 to start with.